app/views/contexts/submission_metadata_context.py

```diff
@@ -26,7 +26,7 @@ def build_submission_metadata_context(
         "term": lazy_gettext("Submission reference:"),
         "descriptions": [{"description": convert_tx_id(tx_id)}],
     }
-    if survey_type is SurveyType.SOCIAL:
+    if survey_type in [SurveyType.SOCIAL, SurveyType.HEALTH]:
```

Comment:
Minor, but use a `set` for `in` operators. List for loops.
```suggestion
    if survey_type in {SurveyType.SOCIAL, SurveyType.HEALTH}:
```